pkg/kubeapiserver/options/authentication.go

```diff
@@ -440,6 +445,10 @@ func (o *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 			"which helps safe transition from legacy token to bound service account token feature. "+
 			"If this flag is enabled, admission injected tokens would be extended up to 1 year to "+
 			"prevent unexpected failure during transition, ignoring value of service-account-max-token-expiration.")
+
+		o.ServiceAccounts.IsMaxExpirationConfigured = func() bool {
+			return fs.Changed("service-account-max-token-expiration")
```

Comment:
I guess if we don't have a default for this value, we can just check if it is `!= 0` rather than plumbing a set flag detector function ... sorry for the churn there, I really thought we had a default, but I guess we don't

Reply:
Done